javagen/src/main/java/com/azure/autorest/template/ServiceClientBuilderTemplate.java

```diff
@@ -122,9 +122,29 @@ public final void write(ServiceClient serviceClient, JavaFile javaFile) {
                     javaBlock.line("this.pipelinePolicies = new ArrayList<>();");
                 });
             }
+
+            // ServiceVersion in builder
+            List<ServiceClientProperty> serviceClientProperties = new ArrayList<>(serviceClient.getProperties());
+            if (settings.isLowLevelClient()) {
+                for (int i = 0; i < serviceClientProperties.size(); i++) {
```

Comment:
nit. Might worth use `ListIterator`, so you can replace the value in-place (without the `set(i, ...)`).

Reply:
https://github.com/Azure/autorest.java/pull/1126#discussion_r681551103

Reply:
This code snippet is removed.